tests/app/celery/test_scheduled_tasks.py

```diff
@@ -89,6 +93,8 @@ def test_should_have_decorated_tasks_functions():
         'switch_current_sms_provider_on_slow_delivery'
     assert delete_inbound_sms_older_than_seven_days.__wrapped__.__name__ == \
         'delete_inbound_sms_older_than_seven_days'
+    assert delete_dvla_response_files_older_than_seven_days.__wrapped__.__name__ == \
+        'delete_dvla_response_files_older_than_seven_days'
```

Comment:
Shouldn't this be trying to ensure that the celery task name `delete_dvla_response_files` is linked to the function? 
I think that this is not implemented correctly as I can change the celery task name and the test still passes leading the test to report false positives.